plugins/api-docs/src/components/AsyncApiDefinitionWidget/AsyncApiDefinition.tsx

```diff
@@ -19,6 +19,7 @@ import '@asyncapi/react-component/styles/default.css';
 import { makeStyles, alpha, darken } from '@material-ui/core/styles';
 import React from 'react';
 import { useTheme } from '@material-ui/core/styles';
+import { AsyncApiResolver } from './AsyncApiDefinitionWidget';
```

Comment:
maybe keep the type here since it belongs to this component and to avoid circular dependencies

```suggestion

/** @public */
export type AsyncApiResolver = {
  schema: string;
  order: number;
  canRead: boolean;
  read(uri: any): Promise<string>;
};
```